.vscode/launch.json

```diff
@@ -51,7 +51,7 @@
       "skipFiles": ["<node_internals>/**"],
       "restart": true,
       "smartStep": false,
-      "outFiles": ["${workspaceRoot}/dist/server.js"]
+      "outFiles": ["${workspaceRoot}/dist/server_node.js"]
```

Comment:
These can be globs, so I guess it could also be `"${workspaceRoot}/dist/server_*.js"`?

Reply:
oh yea that makes sense 👍